src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -728,13 +728,17 @@ void CallTreeWidget::OnCustomContextMenuRequested(const QPoint& point) {
     }
   } else if (action->text() == kActionDisassembly) {
     metrics_uploader_->SendLogEvent(OrbitLogEvent::ORBIT_DISASSEMBLY_REQUESTED);
+    int i = 0;
     for (const FunctionInfo* function : functions) {
       app_->Disassemble(app_->GetCaptureData().process_id(), *function);
+      if (++i > 9) break;
```

Comment:
Please declare this const before and reuse it below.
(`const kMaxNumOpenedWindowsPerOp = 10` - I'm not finding a good name) 

Also please use `>=` instead of `>`

Reply:
done